package.json

```diff
@@ -23,7 +23,7 @@
     "test:coverage": "nyc jasmine --config=tests/spec/coverage.json"
   },
   "dependencies": {
-    "cordova-common": "^3.1.0",
+    "cordova-common": "^3.2.0",
     "electron": "^4.0.1",
     "electron-builder": "^20.38.4",
     "fs-extra": "^7.0.1",
```

Comment:
Should this be bumped, since common now uses 8.0.x?

Reply:
I have a separate PR already in place for that.

https://github.com/apache/cordova-electron/pull/66

Once I merge this PR in, I will re-build the `package-lock.json` on that PR as well.